recipes/wstools/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "wstools" %}
+{% set version = "0.4.5" %}
+{% set sha256 = "499760d87d9bb62fb937669bb9acb2de8a6393dcce3c5fbd3bc693b4a7bdd4ba" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
```

Comment:
0.4.6 release from Github breaks for me with "Exception: Versioning for this project requires either an sdist tarball, or access to an upstream git repository. Are you sure that git is installed?" - this is from `pbr`. I'm busy investigating this.